data/yam/agama/auto/template.libsonnet

```diff
@@ -66,7 +69,13 @@ function(bootloader=true,
             [if scripts_post_partitioning != '' then 'postPartitioning']: [ scripts_post_partitioning_lib[x] for x in std.split(scripts_post_partitioning, ',') ],
             [if scripts_pre != '' then 'pre']: [ scripts_pre_lib[x] for x in std.split(scripts_pre, ',') ],
           },
-          [if decrypt_password != '' then 'questions']: answers_lib.questions_decrypt(decrypt_password),
+          [if decrypt_password != '' || trust_import_gpg then 'questions']: {
+            policy: 'auto',
```

Comment:
in theory this parameter is not needed because there are answer predefined, right? https://agama-project.github.io/docs/user/reference/profile/answers#the-policy-property Please try to remove it if makes sense and provide verification. Otherwise this is not the right place to add it, but in the library.